src/components/User/Users/components/UserList.vue

```diff
@@ -1,137 +1,125 @@
 <template>
   <div class="user-list">
-    <template v-if="view.mode === VIEW_MODES_VERBOSE.index">
-      <div class="user-list__filters-wrp">
-        <div class="app-list-filters">
-          <select-field
-            class="issuance-rl__filter app-list-filters__field"
-            label="Role"
-            v-model="filters.role"
-          >
-            <option :value="''" />
-            <option :value="ACCOUNT_ROLES.notVerified">
-              Unverified
-            </option>
-            <option :value="ACCOUNT_ROLES.general">
-              General
-            </option>
-            <option :value="ACCOUNT_ROLES.corporate">
-              Сorporate
-            </option>
-          </select-field>
-
-          <input-field
-            class="app-list-filters__field"
-            v-model.trim="filters.requestor"
-            label="Requestor"
-            autocomplete-type="email"
-          />
-        </div>
+    <div class="user-list__filters-wrp">
+      <div class="app-list-filters">
+        <select-field
+          class="issuance-rl__filter app-list-filters__field"
```

Comment:
`issuance-rl__filter` is useless

Reply:
I'm not sure about this. Look, .issuance-rl__filter has css style (flex: 0.33) and it's parent has value display: flex.